src/main/java/org/broadinstitute/consent/http/resources/ElectionReviewResource.java

```diff
@@ -1,42 +1,20 @@
 package org.broadinstitute.consent.http.resources;
```

Comment:
Is there something that uses this endpoint?  I didn't see where it was leveraged in the frontend.

Reply:
Good find, I thought there was ... I am mistaken. Removing now!